test_chesapeakecvpr_models.py

```diff
@@ -47,6 +52,13 @@ def set_up_parser() -> argparse.ArgumentParser:
         help="path to the CSV file to write results",
         metavar="FILE",
     )
+    parser.add_argument(
+        "--gpu",
```

Comment:
Should be renamed `--device` like in my other PR, some users may not have a GPU and some systems have multiple CPUs per node.

Reply:
I don't particularly care what this is called, but also it doesn't make sense to run this file without a GPU and I don't think `torch.device("cpu", device)` will actually do anything (I just tested this and it used all cores on my machine anyway).

Reply:
Done